lib/tasks/test_vmdb.rake

```diff
@@ -4,7 +4,7 @@ if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
 namespace :test do
   namespace :vmdb do
     desc "Setup environment for vmdb specs"
-    task :setup => [:verify_no_db_access_loading_rails_environment, :setup_db]
+    task :setup => [:initialize, :verify_no_db_access_loading_rails_environment, :setup_db]
```

Comment:
Particularly as this lives in a different file, if we're going out of our way to run something called "initialize" before 'verify', we should probably have the latter do something to ensure its test is valid (i.e., that the environment hasn't already been loaded somehow). Otherwise it could quietly become a no-op, and we'd lose its protection.